<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setForm</class>
 <widget class="QWidget" name="setForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="1">
    <widget class="QPushButton" name="buttonApply">
     <property name="minimumSize">
      <size>
       <width>75</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>75</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>299</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QFrame" name="frameInfo">
     <property name="minimumSize">
      <size>
       <width>378</width>
       <height>73</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>378</width>
       <height>73</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255,0,0,20)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>5</number>
      </property>
      <item row="0" column="1">
       <widget class="QPushButton" name="btn4CloseInfo">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgba(0,0,0,0)</string>
        </property>
        <property name="text">
         <string>x</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>88</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" rowspan="2">
       <widget class="QTextEdit" name="textInfo">
        <property name="minimumSize">
         <size>
          <width>351</width>
          <height>71</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgba(0,0,0,0)</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="minimumSize">
      <size>
       <width>361</width>
       <height>161</height>
      </size>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="unit">
      <attribute name="title">
       <string>Unit</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Units</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QToolButton" name="btnChange">
         <property name="text">
          <string>chan</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="btnSave">
         <property name="text">
          <string>save</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QToolButton" name="btnAdd">
         <property name="text">
          <string>add</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QToolButton" name="btnDel">
         <property name="text">
          <string>del</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="5">
        <widget class="QTableView" name="tableView">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="hardware">
      <attribute name="title">
       <string>hardware</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="group_antSystem">
         <property name="title">
          <string>ant system</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QComboBox" name="combo_nameAntSystem"/>
          </item>
          <item>
           <widget class="QPushButton" name="btn_addAntSystem">
            <property name="minimumSize">
             <size>
              <width>31</width>
              <height>23</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>31</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_delAntSystem">
            <property name="minimumSize">
             <size>
              <width>31</width>
              <height>23</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>31</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_changeAntSystem">
            <property name="minimumSize">
             <size>
              <width>31</width>
              <height>23</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>31</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string>!</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="group_sender">
         <property name="title">
          <string>sender</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QComboBox" name="combo_nameSender"/>
          </item>
          <item>
           <widget class="QPushButton" name="btn_addSender">
            <property name="minimumSize">
             <size>
              <width>31</width>
              <height>23</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>31</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_delSender">
            <property name="minimumSize">
             <size>
              <width>31</width>
              <height>23</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>31</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_changeSender">
            <property name="minimumSize">
             <size>
              <width>31</width>
              <height>23</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>31</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string>!</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="group_recvier">
         <property name="title">
          <string>recvier</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QComboBox" name="combo_nameRec"/>
          </item>
          <item>
           <widget class="QPushButton" name="btn_addRec">
            <property name="minimumSize">
             <size>
              <width>31</width>
              <height>23</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>31</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_delRec">
            <property name="minimumSize">
             <size>
              <width>31</width>
              <height>23</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>31</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_changeRec">
            <property name="minimumSize">
             <size>
              <width>31</width>
              <height>23</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>31</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string>!</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
